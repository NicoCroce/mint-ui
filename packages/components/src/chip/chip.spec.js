import * as React from "react";
import { render } from "@testing-library/react";
import Chip from "./index";

test("chip renders correctly", () => {
    const { asFragment } = render(<Chip text="November" />);
    expect(asFragment()).toMatchSnapshot();
});