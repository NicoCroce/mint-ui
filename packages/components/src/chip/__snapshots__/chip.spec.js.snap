// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`chip renders correctly 1`] = `
<DocumentFragment>
  <span
    class="chip__Wrapper-icu7z7-0 gyQtLo"
  >
    <span
      class="chip__Text-icu7z7-1 bAmdaL"
    >
      November
    </span>
  </span>
</DocumentFragment>
`;
